README.md

```diff
@@ -114,6 +114,26 @@ Use it at your own risk.
 
 Please see the [LICENSE](https://github.com/pfnet-research/chainer-chemistry/blob/master/LICENSE) file for details.
 
+## Links
+
+Links for Chainer Chemistry:
+
+ - Document: [https://chainer-chemistry.readthedocs.io](https://chainer-chemistry.readthedocs.io)
+ - Blog: [Release Chainer Chemistry: A library for Deep Learning in Biology and Chemistry](https://preferredresearch.jp/2017/12/18/chainer-chemistry-beta-release/)
+
+Links for other Chainer project:
+
+ - Chainer: A flexible framework of neural networks for deep learning - [chainer/chainer](https://github.com/chainer/chainer)
```

Comment:
How about adding a link to titles (Chainer in this case)?

Reply:
Is it better to use the official website (https://chainer.org) as a destination URL?

Reply:
> adding a link to titles
What do you mean?

> Is it better to use the official website (https://chainer.org) as a destination URL?
May be we can put both (official page link as well as github link)